src/esrs/aggregate.rs

```diff
@@ -106,7 +106,7 @@ pub trait AggregateManager: Aggregate {
     /// the aggregate state by order of their sequence number
     ///
     /// You should _avoid_ implementing this function, and be _very_ careful if you decide to do so.
-    async fn load(&self, aggregate_id: impl Into<Uuid> + Send) -> Option<AggregateState<Self::State>> {
+    async fn load(&self, aggregate_id: impl Into<Uuid> + Send + 'async_trait) -> Option<AggregateState<Self::State>> {
```

Comment:
Don't get why is this needed. I'm using rust 1.64.0 and no one complains for this

Reply:
Uhm, that's strange 🤔 

I'll try to deep dive as soon as possible 🙏

Reply:
Local "cache" issue. No esrs issue, my bad 🤦‍♂️